lib/rules.ts

```diff
@@ -18,6 +20,8 @@ export type RunContext = {
   urlPattern?: string,
 }
 
+export type ElementSelector = string | Selector[]
```

Comment:
according to the docs, [Selector](https://github.com/puppeteer/replay/blob/main/docs/api/README.md#selector) is already defined as `string | string[]`, why do we need another wrapper type here?